finetuner/client/client.py

```diff
@@ -0,0 +1,57 @@
+from finetuner.client.base import BaseClient
+from finetuner.client.endpoints import Endpoints
+from finetuner.constants import DELETE, GET, NAME, POST, USER_ID
+
+
+class Client(BaseClient):
+    def create_experiment(self, experiment_name: str):
```

Comment:
i think when you're creating an experiment, the parameter `experiment_name` can be simply `name`. You know you're creating an experiment name anyway. It's like when you designing a database, table name is `User`, but you call the id field `user_id`, which is not necessary. you call it `id` and you know it's belongs to `User.id`.